Sources/Extensions/UIView+Responder.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIView+Responder.swift
+//  FBSnapshotTestCase
+//
+//  Created by Jelle Vandebeeck on 08/05/2018.
+//
+
+import UIKit
+
+extension UIView {
+    /// Return the controller that is manages this view.
```

Comment:
🖕 Typo in the description? `Returns the UIViewController that manages this view`